iris/docs/v2.2.1/whatsnew/2.2.html

```diff
@@ -0,0 +1,281 @@
+discontiguities in the bounds arrays of cube coordinates.  Additionally,
+discontiguous points in coordinates can be explicitly masked
+using another new feature <a class="reference internal" href="../iris/iris/util.html#iris.util.mask_cube" title="iris.util.mask_cube"><code class="xref py py-func docutils literal notranslate"><span class="pre">iris.util.mask_cube()</span></code></a>.</li>
+<li><a class="reference internal" href="../iris/iris/util.html#iris.util.array_equal" title="iris.util.array_equal"><code class="xref py py-func docutils literal notranslate"><span class="pre">iris.util.array_equal()</span></code></a> now has a ‘withnans’ keyword, which provides
+a NaN-tolerant array comparison.</li>
+</ul>
+</div>
+<div class="section" id="iris-2-2-dependency-updates">
+<h2>Iris 2.2 Dependency updates<a class="headerlink" href="#iris-2-2-dependency-updates" title="Permalink to this headline">¶</a></h2>
+<ul class="simple">
+<li>Iris is now using the latest version release of dask (currently 0.19.3)</li>
+<li>Proj4 has been temporarily pinned to version &lt; 5 while problems with the
+Mollweide projection are addressed.</li>
+<li>Matplotlib has been pinned to version &lt; 3 temporarily while we account for
+its changes in all SciTools libraries.</li>
+</ul>
+</div>
+<div class="section" id="bugs-fixed">
+<h2>Bugs Fixed<a class="headerlink" href="#bugs-fixed" title="Permalink to this headline">¶</a></h2>
+<ul class="simple">
+<li>The bug has been fixed that prevented printing time coordinates with bounds
+when the time coordinate was measured on a long interval (that is, <code class="docutils literal notranslate"><span class="pre">months</span></code>
+or <code class="docutils literal notranslate"><span class="pre">years</span></code>).</li>
+<li>“Gracefully filling…” warnings are now only issued when the coordinate or
+bound data is actually masked.</li>
+</ul>
+<div class="section" id="bugs-fixed-in-v2-2-1">
+<h3>Bugs fixed in v2.2.1<a class="headerlink" href="#bugs-fixed-in-v2-2-1" title="Permalink to this headline">¶</a></h3>
+<ul class="simple">
+<li>Iris can now correctly unpack a column of header objects when saving a
+pandas DataFrame to a cube.</li>
+<li>fixed a bug in <a class="reference internal" href="../iris/iris/util.html#iris.util.new_axis" title="iris.util.new_axis"><code class="xref py py-meth docutils literal notranslate"><span class="pre">iris.util.new_axis()</span></code></a> : copying the resulting cube
+resulted in an exception, if it contained an aux-factory.</li>
+<li><a href="#id1"><span class="problematic" id="id2">:class:`iris.coords.AuxCoord`_</span></a>’s can now test as ‘equal’ even when the points
+or bounds arrays contain NaN values, if values are the same at all points.
+Previously this would fail, as conventionally “NaN != NaN” in normal
+floating-point arithmetic.</li>
```

Comment:
It's also worth being aware of the built html copy of the whats new.
You can see that the extra `_` caused an issue for the html see line 224:
```
<li><a href="#id1"><span class="problematic" id="id2">:class:`iris.coords.AuxCoord`_</span></a>’s can now test as ‘equal’ even when the points```